protocol/contracts/farm/facets/SeasonFacet/Sun.sol

```diff
@@ -112,16 +116,15 @@ contract Sun is Oracle {
             .add(amount);
     }
 
-    function setSoilAbovePeg(uint256 newHarvestable, uint256 caseId) internal {
-        uint256 newSoil = newHarvestable.mul(100).div(100 + s.w.yield);
-        if (caseId >= 24) newSoil = newSoil.mul(C.soilCoefficientHigh()).div(C.precision());
-        else if (caseId < 8) newSoil = newSoil.mul(C.soilCoefficientLow()).div(C.precision());
-        setSoil(newSoil);
+    function setSoilAndPeasAbovePeg(uint256 newHarvestable, uint256 caseId) internal {
+        uint256 maxPeas = newHarvestable;
```

Comment:
Why not just rename `newHarvestable` to `maxPeas` in the function signature instead of instantly creating a new variable called `maxPeas`?

Reply:
good point, removed `maxPeas`